components/demo/code-view.js

```diff
@@ -36,10 +36,6 @@ class CodeView extends LitElement {
 		super.attributeChangedCallback(name, oldval, newval);
 	}
 
-	firstUpdated() {
```

Comment:
This was causing the duplicate rendering of `d2l-code-view`. The `slotchange` event already handles updating the code.